src/javascript/app/ng-wig/ng-wig.component.js

```diff
@@ -58,7 +58,13 @@ angular.module('ngWig')
           return;
         }
 
-        let pasteContent = (event.originalEvent || event).clipboardData.getData('text/plain');
+        let pasteContent;
+        if (window.clipboardData && window.clipboardData.getData) { // IE  
```

Comment:
are you sure about such check? 

    window.clipboardData


may be we could go with this solution?
http://stackoverflow.com/a/6804718/274500

     if (/text\/html/.test(event.clipboardData.types))